doozerlib/cli/images_upstreampulls.py

```diff
@@ -0,0 +1,62 @@
+# stdlib
+import os
+import yaml
+
+# external
+from github import Github
+
+# doozerlib
+from doozerlib import constants
+from doozerlib.cli import cli, pass_runtime
+
+github_token = os.getenv(constants.GIT_TOKEN)
```

Comment:
Git != GitHub. There is no such thing called `GIT_TOKEN`. Would like to rename it to something like `GITHUB_TOKEN`.